src/_data/side-nav.yml

```
@@ -165,7 +165,7 @@
           permalink: /tutorials/web/fetch-data
         - title: Low-level web programming
           children:
-          - title: Connect Dart & HTML
+          - title: Connect Dart and HTML
```

We intentionally use & in the sidenav and tabs to save space. Might as well be consistent.

```suggestion
          - title: Connect Dart & HTML
```